src/actions/index.js

```diff
@@ -66,13 +80,18 @@ export const logIn = (login, password) => dispatch => fetch('http://localhost:30
     body: JSON.stringify({ login, password }),
   })
   .then(res => res.json())
-  .then((res) => { if (res.key) dispatch(logInSucces()); })
+  .then((res) => {
+    if (res.token) {
+      localStorage.setItem('token', res.token);
+      dispatch(logInSucces());
+    }
+  })
   .catch(err => dispatch(errorAfterFetch(err)));
 
-export const logOut = () => ({
-  type: 'LOGOUT',
-  isAuthenticated: false,
-});
+export const logOut = () => (dispatch) => {
+  localStorage.removeItem('token');
```

Comment:
In real life scenario we would have to make a logout call to the server to remove the token from the user.

Check https://trello.com/c/7p10ILk1/37-validation-via-json-server